cmd/geth/consolecmd.go

```diff
@@ -174,6 +150,15 @@ func remoteConsole(ctx *cli.Context) error {
 	return nil
 }
 
+// ephemeralConsole starts a new geth node, attaches an ephemeral JavaScript
+// console to it, executes each of the files specified as arguments and tears
+// everything down.
+func ephemeralConsole(ctx *cli.Context) error {
+	utils.Fatalf(`The "js" command is deprecated. Please use the following instead:
```

Comment:
If we want to be user-friendly: 
```suggestion
	var b strings.Builder
	for _, file := range ctx.Args() {
		b.Write([]byte(fmt.Sprintf("loadScript('%s');", file)))
	}
	utils.Fatalf(`The "js" command is deprecated. Please use the following instead:
geth --exec "%s" console`, b.String())
```

Reply:
So... Any thoughts on my suggestion there ^ ?